testgrid/tgrun/pkg/scheduler/scheduler.go

```diff
@@ -107,11 +113,42 @@ func Run(schedulerOptions types.SchedulerOptions) error {
 		return errors.Wrap(err, "failed to report ref started")
 	}
 
-	fmt.Printf("Started tests on %d specs across %d images\n", len(instances.Instances), len(operatingSystems))
+	fmt.Printf("Started tests on %d specs across %d images\n", len(kurlPlans), len(operatingSystems))
 
 	return nil
 }
 
+func getKurlPlans(schedulerOptions types.SchedulerOptions) ([]types.Instance, error) {
+	var kurlPlans []types.Instance
+
+	// Custom Kurl Spec takes precedence
+	if schedulerOptions.Spec != "" {
```

Comment:
is tgapi open? last time i checked i think it was. i worry that this is a security issue as it may open us up to arbitrary specs being run. right now we dont have anything like addon `script` or `helm` but if we did wouldnt this be an issue? @laverya wanna weigh in here?